managed_vms/endpoints/src/main/appengine/swagger.yaml

```diff
@@ -0,0 +1,106 @@
+swagger: "2.0"
```

Comment:
Should also mention all the places you need to edit to make it work.

**host**, **YOUR-CLIENT-ID**, etc.